tsinfer/formats.py

```diff
@@ -2333,6 +2356,26 @@ def sequence_length(self):
     def num_sites(self):
         return self._num_sites
 
+    @functools.cached_property
+    def individuals_mask(self):
+        if self._sgkit_samples_mask_name is None:
+            return np.full(self._num_unmasked_individuals, True, dtype=bool)
+        else:
+            try:
+                # We negate the mask as it is much easier in numpy to have True=keep
+                return ~(self.data[self._sgkit_samples_mask_name][:].astype(bool))
```

Comment:
It's pretty confusing here to have ``mask`` being used in both the positive and negative sense - so an input mask is False to keep, but the masks we use internally are True to keep. I think we need to use different words here.

So, a mask is a boolean array which is true if the corresponding item should be *excluded*.

How about a "select" array is boolean array which is true if the corresponding item should be kept. Then, ``x_select`` and ``x_mask`` arrays are logical inverses.

Reply:
Yeah, good point.

Reply:
Yeah, great call @jeromekelleher . I would find this much easier to follow too.